src/irq.rs

```diff
@@ -0,0 +1,8 @@
+use axerrno::AxResult;
+use axvcpu::AxVCpuExitReason;
+
+use crate::TrapFrame;
+
+pub fn exception_handle_irq(_ctx: &mut TrapFrame) -> AxResult<AxVCpuExitReason> {
```

Comment:
Add comments on this function, explain why you fix the vector as 33.

Add something like:

```Rust
// Todo: 
```

Reply:
ok